guides/common/modules/proc_renewing-a-custom-ssl-certificate-proxy.adoc

```diff
@@ -0,0 +1,91 @@
+[id="Renewing_a_Custom_SSL_Certificate_Proxy_{context}"]
+= Renewing a {customssltitle} certificate on {SmartProxyServer}
+
+Use this procedure to renew a {customssl} certificate signed by a Certificate Authority for your {SmartProxyServer}.
+The `{foreman-installer}` command, which the `{certs-generate}` command returns, is unique to each {SmartProxyServer}.
+Do not use the same command on more than one {SmartProxyServer}.
+
+.Prerequisites
+Before renewing a custom server certificate for {SmartProxyServer}, ensure that your {Project} and {SmartProxies} meet the following conditions:
+
+* {ProjectServer} is configured with a custom certificate.
+* {SmartProxyServer} is registered to {ProjectServer}.
+* The {SmartProxyServer} packages are installed.
```

Comment:
I think we could drop this, if we are renewing certificates then the assumption stated or otherwise is that the Smart-proxy server is installed correctly and operating connected to the Foreman/Satellite.

Reply:
Addressed :+1: